### PR TITLE
[FIX] website_crm_recaptcha: Make uninstall work

### PR DESCRIPTION
**Pull request by yajo**
Without this patch, after uninstalling the module, all website_crm forms will fail.

@Tecnativa

### CODE REVIEW

File: website_crm_recaptcha/__manifest__.py [hunk 1]
```diff
@@ -12,15 +12,16 @@
...
+    "uninstall_hook": "uninstall_hook",
     "depends": [
         "website_crm",
         'website_form_recaptcha',
     ],
     "data": [
-        "data/ir_model_data.xml",
-        'views/website_crm_template.xml',
+        "data/ir_model_data.xml",'
+        'te.xml',
```
Comment (pedrobaeza) [inline website_crm_recaptcha/__manifest__.py]: ?

File: website_crm_recaptcha/__manifest__.py [hunk 2]
```diff
@@ -12,15 +12,16 @@
...
     ],
-    'images': [
-        'static/description/website_crm_recaptcha.jpg',
+    'images': [' \
+        '/description/website_crm_recaptcha.jpg',
```
Comment (pedrobaeza) [inline website_crm_recaptcha/__manifest__.py]: ?

File: website_crm_recaptcha/__manifest__.py (hunk 1 above)
Comment (yajo) [reply website_crm_recaptcha/__manifest__.py]: Oops, some bug in vscode or some extension... Fixed.

**Review by pedrobaeza — APPROVED** [review]
Fast-tracking this as straight-forward.

**Comment by pedrobaeza** [comment]
Is this applicable to v11?

/ocabot merge patch

**Comment by yajo** [comment]
> Is this applicable to v11?

Yes.

**Comment by pedrobaeza** [comment]
Cherry-picked for v11 in 8761e601ee4c774415a1c6b229c7e89fb9db7e8d

**Pull request closed without merging**